package.json

```diff
@@ -75,7 +75,7 @@
     "solc": "^0.8.3"
   },
   "dependencies": {
-    "@solidity-parser/parser": "^0.12.1",
+    "@solidity-parser/parser": "^0.12.2",
```

Comment:
the package.json already had v0.12.2 btw

Reply:
yeah, dependabot only changes the lock files if it can help it